### PR TITLE
Topo device service - watchTopoEntity

### PR DESCRIPTION
**Pull request by rgemawat2000**
I'm not sure what the protocolList is for devices, and it does not seem to exist for entities, so should I delete the commented lines?
Another thing is, I have left this as a draft PR for now because I was not sure whether to call watchTopoEntity in device list component or if I should create the new topo entity component in this PR

### CODE REVIEW

File: web/onos-gui/src/app/onos-topo/topodevice.service.ts [hunk 1]
```diff
@@ -32,6 +34,7 @@ import { KeyValue } from '@angular/common';
 export class TopoDeviceService {
     deviceList: Map<string, Device>; // Expect <dev-id:dev-ver> as key
     topoDevicesSub: Subscription;
```
Comment (SeanCondon) [inline web/onos-gui/src/app/onos-topo/topodevice.service.ts:36]: you probably should create a `topoEntitySub : Subscription` too - it's there so we can have a handle on it when we need to close it

File: web/onos-gui/src/app/onos-topo/topodevice.service.ts [hunk 2]
```diff
@@ -260,6 +264,71 @@ export class TopoDeviceService {
...
+                    //         protocol.getServicestate() !== newProtoState.getServicestate()) {
+                    //         this.entityList.set(name, resp.getUpdate().getObject().getEntity());
+                    //     }
+                    // });
+                    // In case there are some in the new set that are not in the old - just push them
+                    // updated.getProtocolsList().filter((p) => !protosHandled.includes(p.getProtocol())).forEach((p) => {
+                    //     this.entityList.set(name, resp.getUpdate().getObject().getEntity());
+                    // });
+                } else {
+                    console.log('Unhandled Topo update', resp.getUpdate().getType(), resp.getUpdate().getObject().getId());
```
Comment (SeanCondon) [inline web/onos-gui/src/app/onos-topo/topodevice.service.ts:304]: maybe call the `errorCb` here

Comment (rgemawat2000) [reply web/onos-gui/src/app/onos-topo/topodevice.service.ts:304]: okay will do, I am also adding a stopWatchingTopoEntity function

File: web/onos-gui/src/app/onos-topo/topodevice.service.ts (hunk 1 above)
Comment (rgemawat2000) [reply web/onos-gui/src/app/onos-topo/topodevice.service.ts:36]: okay

**Comment by rgemawat2000** [comment]
will make a seperate path for duplicating deviceList component

**Comment by SeanCondon** [comment]
i'm going to merge this if that's OK

**Comment by rgemawat2000** [comment]
yes, looks good!

**Comment by rgemawat2000** [comment]
I'll get started on the new component now

**Pull request merged**